sciencer/utils/csv_callback.py

```diff
@@ -0,0 +1,48 @@
+"""WriteToCSV Callbacks
+"""
+import csv
+from typing import Dict
+
+from sciencer.core import Callbacks
+from sciencer.models import Paper
+
+
+def convert_to_row(paper: Paper) -> Dict[str, str]:
+    """Converts a paper into a dictionary
+
+    Args:
+        paper (Paper): paper to conver
+
+    Returns:
+        Dict[str, str]: paper's properties
+    """
+    row = {}
+    row['title'] = paper.title if paper.title is not None else ""
+    row['year'] = str(paper.year) if paper.year is not None else ""
+    row['authors_s2_id'] =  "["+''.join([f"{author_id};" for author_id in paper.authors])[:-1]+"]"
+    return row
+
+field_names = ['title', 'authors_s2_id', 'year', 'accepted']
+
+class WriteToCSVCallbacks(Callbacks):
+    """Class that writes to a csv file

```

Comment:
I would suggest a comment on how the CSV would be structured. But this is optional.